Code-250/my-brand
# VoFBnJwZ-fix-navigations and elements positionings

### What does this PR do?

This pull request fixes the navigations on the landing page, login page, and admin pages with their responsiveness

### Description of Task to be completed?

- should resolve all navigations as they are in figma design
- should resolve every problem that is in the codes bot positionings and any other
- should scroll to the section clicked on the navigation bar of the landing page
- add skill models on the contact page
- make responsive all admin pages and models

### How should this be manually tested?

- `git clone to have the repository to your local machine
- `git checkout fix-navigations-VoFBnJwZ` to go to the branch that you can use to test this.
- run `index.html ` in your browser
- Navigate to all different navigation links and buttons to explore how they work.
- make sure that your computer is on normal zoom which is 100%.

### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

- [fix-navigatio](https://trello.com/c/VoFBnJwZ/18-fix-navigations)

### Screenshots (if appropriate)
- desktop
![image](https://user-images.githubusercontent.com/47631065/147613073-09f114f4-b209-4364-ac10-a1f698af9f6d.png)


- mobile
![mobile](https://user-images.githubusercontent.com/47631065/147049383-7cc74671-110c-4176-a968-5fd4a65c3f67.png)

- mobile of blogsadmin dashboard
![image](https://user-images.githubusercontent.com/47631065/147825605-eae17b75-bcc3-43f4-8ec0-2b71aa0b823d.png)

- mobile of contact querries admin dashboard
![image](https://user-images.githubusercontent.com/47631065/147825553-5480663e-ef6a-4cb7-bd13-2e0ae93919ec.png)

### Questions:
N/A
